src/classes/LD_LeadConvertOverride_CTRL.cls

```diff
@@ -375,10 +375,9 @@ public with sharing class LD_LeadConvertOverride_CTRL {
                 lc.setAccountID(CAO_Constants.getIndividualAccountID());
 	        }
 
-	        // avoid auto-affiliation creation
-            AFFL_Affiliations_TDTM.doNotRunForContacts = true;
-            AFFL_Affiliations_TDTM.hasRunForAccounts = true;
-            
+	        // avoid auto-affiliation creation by disabling the Affiliations Trigger for Contacts
+            TDTM_TriggerHandler.toggleTriggerState('Contact', 'AFFL_Affiliations_TDTM', false);
```

Comment:
why turning off the trigger directly, rather than continuing to use flags/counters?